### PR TITLE
Add `System.global` tests

### PR DESCRIPTION
**Pull request by ljharb**
(https://github.com/tc39/proposal-global/issues/12)

Since this proposal is stage 2, it's OK to hold off on merging these tests - I will seek stage 3 in May after which it would be ideal to have them merged :-)

### CODE REVIEW

File: test/built-ins/System/System.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: system
```
Comment (leobalter) [inline test/built-ins/System/System.js]: I believe this should be `esid: sec-system` or if the id is not defined yet we can use `esid: pending`.

File: test/built-ins/System/global/property-descriptor.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+/*---
+esid: sec-system.global
+description: System.global should be non-writable, non-enumerable, and configurable
+author: Jordan Harband
+includes: [propertyHelper.js]
+---*/
+
+verifyNotEnumerable(Object, 'values');
+verifyNotWritable(Object, 'values');
+verifyConfigurable(Object, 'values');
```
Comment (leobalter) [inline test/built-ins/System/global/property-descriptor.js]: s/Object.values/System.global

File: test/built-ins/System/System.js (hunk 1 above)
Comment (ljharb) [inline test/built-ins/System/System.js]: good call, will fix

File: test/built-ins/System/global/property-descriptor.js (hunk 2 above)
Comment (ljharb) [inline test/built-ins/System/global/property-descriptor.js]: lulz, oops

**Comment by jugglinmike** [comment]
@ljharb This may be of interest of you (for some stretched definition of the word, "interest"): https://github.com/tc39/test262/issues/568

**Comment by ljharb** [comment]
@jugglinmike thanks, definitely some form of interest :-)

Any changes I need to make on this PR specifically? :-D

**Comment by jugglinmike** [comment]
Nope, this all looks good to me! If I learn something new about `fnGlobalObject.js`, I'll let you know

**Comment by leobalter** [comment]
Merged at 45e5fc8. First time I used Github's squash and merge button, and it worked just fine! :+1:

**Comment by ljharb** [comment]
Thanks!

**Comment by littledan** [comment]
I thought TC39's policy was to wait for Stage 3 before merging test262 tests. This proposal is still at Stage 2. Why was it merged? Maybe it should be backed out for now until it reaches Stage 3.

**Comment by leobalter** [comment]
My bad. I'm reverting this and setting a PR to re-add when it's ok.

**Comment by ljharb** [comment]
@littledan there's no policy that things *must* not be merged before stage 3 - only that they must be merged in stage 3.

That said, I'm fine with a revert if you think that's important.

**Comment by goyakin** [comment]
@ljharb Spec changes aren't quite stable yet in Stage 2, so we don't accept tests before Stage 3.

**Comment by goyakin** [comment]
Thanks @leobalter for reverting the tests by the way.

**Comment by ljharb** [comment]
This has now been preempted by #765.

**Pull request merged**